src/DIRAC/WorkloadManagementSystem/Agent/StatesAccountingAgent.py

```diff
@@ -36,24 +40,29 @@ class StatesAccountingAgent(AgentModule):
     __summaryValueFieldsMapping = ["Jobs", "Reschedules"]
     __renameFieldsMapping = {"JobType": "JobSplitType"}
 
+    # PilotsHistory fields
+    __pilotKeyFields = ["TaskQueueID", "GridSite", "GridType", "Status"]
+    __pilotValueFields = ["Pilots"]
+
     def initialize(self):
         """Standard initialization"""
         # This agent will always loop every 15 minutes
         self.am_setOption("PollingTime", 900)
 
         self.backends = self.am_getOption("Backends", "Accounting").replace(" ", "").split(",")
-        messageQueue = self.am_getOption("MessageQueue", "dirac.wmshistory")
+        self.monitoringEnabled = Operations().getValue("monitoringEnabled", "False")
 
-        self.log.info("Committing to %s backend" % "and ".join(self.backends))
```

Comment:
I have moved it to [L110](https://github.com/DIRACGrid/DIRAC/blob/e5fefa1b388c04851e608fec50d17c96db6ff475/src/DIRAC/WorkloadManagementSystem/Agent/StatesAccountingAgent.py#L110), since now we are pushing both pilot and WMS History, and the log would be confusing to which one that line is referring to.